manim_slides/slide.py

```diff
@@ -54,6 +54,14 @@ def __init__(
         self.__loop_start_animation: Optional[int] = None
         self.__pause_start_animation = 0
 
+    @property
+    def __resolution(self) -> Tuple[int, int]:
+        """Returns the scene's resolution used during rendering."""
+        if MANIMGL:
+            return self.camera_config["pixel_width"], self.camera_config["pixel_height"]
+        else:
+            return config["pixel_width"], config["pixel_height"]
+
```

Comment:
@Fairlight8 Sixth, create a new slide property that tells the resolution

Reply:
I don't get why you have to add yet another property here.

Reply:
You don’t « have to ». But as the definition depends on wether Manim or ManimGL is used, I prefer to hide the code behind a property :)

Reply:
Oh, okay, I understand now. No worries, that was silly, I thought we still were in present.py. I understand now.